.github/workflows/release.yml

```diff
@@ -388,13 +388,14 @@ jobs:
     # vcpkg dating from roughly 01/01/2021 or later.
     - name: Update vcpkg
       run: |
-        cd /tmp
-        git clone https://github.com/Microsoft/vcpkg
+        cd /usr/local/share/vcpkg
+        ls -A1 | xargs rm -rf
```

Comment:
why not just go into `/usr/local/share` and do `rm -rf vcpkg` ? After that you can clone it again with `vcpkg` instead of `.`?

Reply:
Because I think that's what I did originally (when first doing the arm64 branch) and the permissions on /usr/local/share didn't allow me to. I suppose I could do a `sudo rm -rf` etc then recreate it, but this way ultimately worked out a bit simpler and avoided `sudo`.

https://github.com/actions/virtual-environments/blob/main/images/macos/provision/core/vcpkg.sh shows the chmod in opreation.

That said, I see now the repository there just had a shallow clone, so perhaps a `git fetch --unshallow` is all I need to do instead...

Reply:
Yes, replaced now with a `git fetch` which looks to be working.